nuxt/pages/HomePage.vue

```diff
@@ -134,6 +159,52 @@ export default Vue.extend({
     popularTracks(): Array<Track> | null {
       return this.tracks ? this.tracks.slice(0, POPULAR_ENTITIES_LIMIT) : null;
     },
+
+    saveTracksPlayable(): Array<Track> {
+      if (!this.tracks) {
+        return [];
+      }
+
+      return this.tracks.filter((track) => this.hasAudioFile(track));
+    },
+  },
```

Comment:
I don't think I want this on the home page to have a `play all` button. Not yet.